### DISCUSSION TITLE
trace  configuring

### DISCUSSION THREAD

**Discussion by veejayvb** [apache/skywalking#11525] [General]
how to configure and where to view documentation for creating traces and send them to skywalking oap servers

**Comment by wu-sheng** [comment]
What do you mean? Agent is automatically creating and sending.

**Comment by veejayvb** [comment]
I have nodejs sample application I need to use skywalking oap and ui components to monitor it, nodejs agent from skywalking states that to use skywalk-backend-js from that agent.start ={ //configuration for skywalking oap server port } is this totally enough to send traces of my nodejs application to oap server

**Reply by wu-sheng** [reply]
What is the question? It seems it is working well alright?
